blueprints/hr_assistant/domains/date/get_date/train.txt

```diff
@@ -0,0 +1,203 @@
+How long has {Jyoti Lajiri|name} been working here?
+What was the date {Brandon LeBlanc|name} {start date|employment_action}
+{date of birth|dob} of {Wilson Adinolfi|name}
+Please get me {DOB|dob} of {Julissa Hunts|name}
+How long has {Simon Roup|name} {worked here|employment_action} and was it the same month that she was {hired|employment_action}?
+Give me the {bday month|dob} of {Ludwick Harrell|name}
+What is {Bonalyn|name}s number of years {of experience|employment_action} at this company?
+Give me the {bday month|dob} of {Ivan Rogers|name}
+When did {Eric|name} {start|employment_action} working at the company?
+in what month was {Maryellen Jackson|name} {born|dob}
+When was {Caitrin Strong|name} {date of hire|employment_action}?
+How long has {Nina Panjwani|name} been with the company?
+Which month does {Simon Roup|name} have his {bday|dob}
+{mia|name} has {been with the company|employment_action} for how long exactly?
+What was {Sneha Jhaveri|name}'s number of {years of experience|employment_action}?
+{date of ending employment|employment_action}, {Anthony Cisco|name} what was it
+whats {Julia Soto|name} num yrs {of experience|employment_action}?
+{date of birth|dob} of {Joelle Burke|name}
+When {April Evensen|name} decided to {end her employment|employment_action}, what was the date
+Give me the {bday month|dob} of {Tyrone Steans|name}
+in what month was {Nan|name} {born|dob}
+{Adell Saada|name} {birth date|dob}
+in what month was {Bartholemew Khemmich|name} {born|dob}
+{nan|name} {birth date|dob}
+What year was {Lily DiNocco|name} {let go|employment_action} of?
+Please get me {DOB|dob} of {mia brown|name}
+The date was what when {Phil Close|name} {been here|employment_action} at the company
+employment {termination date|employment_action} {Amy|name}
+When {Constance Sloan|name} {started|employment_action}, what was the date
+{Sam Athwal|name}'s num yrs {of experience|employment_action}?
+{Amy|name} {date of birth|dob}
+{ended her employment|employment_action}, {Earnest Hankard|name} when
+Can you pl tell me how long has {Jereivan rogersh|name} worked in the firm
+What was the {Amy|name}'s {date of hire|employment_action}?
+Can you please tell me how long has {april evensen|name} worked in the firm
+Give me {Adil Sahoo|name}'s total time he has been {working here for|employment_action}
+Which date was {Randall Pearson|name}'s {join|employment_action} date?
+When did we {fire|employment_action} {Jeff|name}?
+What was {Anna Von Massenbach|name}'s {hiring|employment_action} date?
+{Brigit McCarthy|name} {date of birth|dob}
+whut is {Debbie|name}s {date of birth|dob}?
+{Joelle Burke|name} {bday month|dob}
+Which month does {Taisha Goble|name} have his {bday|dob}
+{DOB|dob} of {Evelyn Girifalco|name}
+{Lori Roby|name}'s {bday|dob} is in which month
+{DOB|dob} of {Karen Mancuso|name}
+{Christie Harrington|name} {start date|employment_action}, what was the date?
+How long has {Giovanni|name} been with the company?
```

Comment:
{been with the company|employment_action}?
Seems like this is a valid entity just three queries below